cocotb/drivers/amba.py

```diff
@@ -284,6 +296,10 @@ def _write_data(self):
                     self._memory[_st:_end] = array.array('B', word.get_buff())
                     burst_count -= 1
                     if burst_count == 0:
+                        self.bus.BVALID.value = 1
+                        while self.bus.BREADY == 0:
```

Comment:
Please use ``<=`` here (and two lines down) to stay consistent with the rest of the code.